app/models/company.rb

```diff
@@ -3,9 +3,12 @@ class Company < ActiveRecord::Base
 
   has_many :opportunities, dependent: :destroy
 
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/black.jpg"
```

Comment:
See the dimensions where it has "300x300>" and "100x100>" make sure that instead of the ">" have a "#", so for e.g "300x300#". This is force paperclip to crop the images to exactly them dimensions.